schemas/test/en/test_progress_value_source_repeating_sections.json

```diff
@@ -0,0 +1,374 @@
+                                        },
+                                        "type": "Date"
+                                    }
+                                ],
+                                "guidance": {
+                                    "contents": [
+                                        {
+                                            "description": "For example 31 12 1970"
+                                        }
+                                    ]
+                                },
+                                "id": "date-of-birth-question",
+                                "title": {
+                                    "placeholders": [
+                                        {
+                                            "placeholder": "person_name_possessive",
+                                            "transforms": [
+                                                {
+                                                    "arguments": {
+                                                        "delimiter": " ",
+                                                        "list_to_concatenate": [
+                                                            {
+                                                                "source": "answers",
+                                                                "identifier": "first-name"
+                                                            },
+                                                            {
+                                                                "source": "answers",
+                                                                "identifier": "last-name"
+                                                            }
+                                                        ]
+                                                    },
+                                                    "transform": "concatenate_list"
+                                                },
+                                                {
+                                                    "arguments": {
+                                                        "string_to_format": {
+                                                            "source": "previous_transform"
+                                                        }
+                                                    },
+                                                    "transform": "format_possessive"
+                                                }
+                                            ]
+                                        }
+                                    ],
+                                    "text": "What is <em>{person_name_possessive}</em> date of birth?"
+                                },
+                                "type": "General"
+                            }
+                        },
+                        {
```

Comment:
I don't think this is working at the moment, when I answer the question in the first section I never see the random question

Reply:
Fixed!